pymc_marketing/plot.py

```diff
@@ -469,7 +470,9 @@ def plot_hdi(
         Figure and the axes
```

Comment:
I know this was not changed in this PR, but the parameters listed above are for `plot_samples()`, and not for `plot_hdi()`.

Reply:
I was trying to see, if the default value of `hdi_prob` is listed. After digging, it defaults to `az.rcParams["stats.ci_prob"]`, which is `0.94`.